### PR TITLE
Drop Ruby 2.5 and Rails 5.0, 5.1 support

### PR DESCRIPTION
**Pull request by mervync**

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -7,16 +7,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.5', '2.6', '2.7', '3.0', '3.1']
-        rails: ['5.0', '5.1', '5.2', '6.0', '6.1', '7.0']
+        ruby: ['2.6', '2.7', '3.0', '3.1']
```
Comment (grosser) [inline .github/workflows/actions.yml:10]: can add 3.2 too

Comment (grosser) [reply .github/workflows/actions.yml:10]: not sure what classic is on, but ideally drop 2.6 🤞

Comment (mervync) [reply .github/workflows/actions.yml:10]: indeed, classic [is the only user](https://cs.github.com/?scopeName=All+repos&scope=&q=user%3Azendesk+arsi+path%3AGemfile.lock)

Comment (grosser) [reply .github/workflows/actions.yml:10]: 🎉

**Comment by mervync** [comment]
will combine these into a single PR...

**Pull request closed without merging**